### DISCUSSION TITLE
Add a detailed monitor option to track GitHub Actions stats

### DISCUSSION THREAD

**Discussion by oscard0m** [community/community#13071] [Actions] [answered]
It would be awesome if GitHub offered per organization/user and at repository level historical stats of GitHub Actions: 
* execution times per
   * workflow
   * job
   * step
* failure/success rate
* etc.

---

On my research, I come up with [Meercode](https://github.com/meercodeio).

I would consider contacting them since they are already offering a [solution](https://github.com/marketplace/meercode-ci-monitoring) with a GitHub App:

![image](https://user-images.githubusercontent.com/2574275/159157465-4dfcdb0a-38aa-4dba-b36c-c58599346512.png)
![image](https://user-images.githubusercontent.com/2574275/159157471-c8ea787e-6818-457d-b639-2ad6afcdad3b.png)
![image](https://user-images.githubusercontent.com/2574275/159157480-68d77426-a466-49ff-8ad8-94ad81729acf.png)

**Comment by ethomson** [comment, marked as answer]
Thanks for the feedback @oscard0m - I agree that this is an area that we can improve. I'll make sure that this is on our backlog. Thanks again.

**Comment by ljubon** [comment]
Is there any update on this request? @ethomson 
It would be good to have some ETA or is it part of road-map 🤞

**Comment by cseas** [comment]
Is there any existing solution at the moment for finding the success rate of a GitHub Action in a repository?

**Comment by ssbarnea** [comment]
https://meercode.io/ seems offline....
